packages/eslint-plugin/src/configs/react-northstar.js

```diff
@@ -3,8 +3,6 @@ module.exports = {
     'airbnb',
     // Extended configs are applied in order, so these configs that turn other rules off should come last
     'prettier',
-    'prettier/@typescript-eslint',
```

Comment:
per [these release notes](https://github.com/prettier/eslint-config-prettier/blob/main/CHANGELOG.md#version-800-2021-02-21) the eslint-config-prettier configs have been merged into one